### PR TITLE
Updates for USWDS v3 color adoption

### PR DESCRIPTION
**Pull request by powellkerry**
## Summary
Update references to v1 colors.
Update formation, css-library, and component-library versions

## Related issue(s)
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2325

## Testing done

- Local visual/inspector testing

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

| Before | After |
|-------|-------|
|localhost_3002_ (8)|localhost_3002_ (9)|
|localhost_3002_find-locations_ (2)|localhost_3002_find-locations_ (3)|
|localhost_3002_life-insurance_ (1)|localhost_3002_life-insurance_ (2)|

## What areas of the site does it impact?

Minimal visual changes to non-blue colors throughout the site

### Quality Assurance & Testing

- [x ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ x] Linting warnings have been addressed
- [x ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ x] Screenshot of the developed feature is added
- [x ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ x] Browser console contains no warnings or errors.
- [x ] Events are being sent to the appropriate logging solution
- [ x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/facility-locator/utils/mapHelpers.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const reverseGeocodeBox = (bounds, types = 'address,postcode') => {
 };
 
 export const staticMapURL = (lat, long, mapboxToken) =>
-  `https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/pin-l+e31c3d(${long},${lat})/${long},${lat},16/500x300?access_token=${mapboxToken}`;
+  `https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/pin-l+d83933(${long},${lat})/${long},${lat},16/500x300?access_token=${mapboxToken}`;
```
Comment (micahchiang) [inline src/applications/facility-locator/utils/mapHelpers.js:95]: is this an actual hexcode update or something different?

File: src/applications/representative-search/utils/mapHelpers.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ export const reverseGeocodeBox = (bounds, types = 'address,postcode') => {
 };
 
 export const staticMapURL = (lat, long, mapboxToken) =>
-  `https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/pin-l+e31c3d(${long},${lat})/${long},${lat},16/500x300?access_token=${mapboxToken}`;
+  `https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/pin-l+d83933(${long},${lat})/${long},${lat},16/500x300?access_token=${mapboxToken}`;
```
Comment (micahchiang) [inline src/applications/representative-search/utils/mapHelpers.js:95]: same question as above

**Review by brianseek — DISMISSED** [review]
Check-in changes look good. I totally forgot we were testing color!

File: src/applications/facility-locator/utils/mapHelpers.js (hunk 1 above)
Comment (powellkerry) [reply src/applications/facility-locator/utils/mapHelpers.js:95]: I researched this and it looks like this is how you tell mapbox what color the pin should be

**Review by eselkin — DISMISSED** [review]
Looks good to me.

**Review by micahchiang — DISMISSED** [review]
Nice work here @powellkerry

**Review by CBonade — DISMISSED** [review]

File: src/applications/check-in/pre-check-in/tests/e2e/pages/Introduction.js
Comment (powellkerry) [inline src/applications/check-in/pre-check-in/tests/e2e/pages/Introduction.js:1]: Cypress tests fail when updating `platform/testing` to `@department-of-veterans-affairs/platform-testing`

File: src/applications/virtual-agent/components/webchat/WebChat.jsx [hunk 3]
```diff
@@ -90,6 +91,7 @@ const WebChat = ({
     conversationId = sessionStorage.getItem(CONVERSATION_ID_KEY);
   }
 
+  // eslint-disable-next-line no-restricted-globals
```
[the remark this answers is not in the document]
Comment (powellkerry) [reply src/applications/virtual-agent/components/webchat/WebChat.jsx:94]: Don't know if there is a better solution here?

**Review by CBonade — APPROVED** [review]

**Comment by humancompanion-usds** [comment]
@powellkerry - I reviewed and all looks good. 

The only thing that stood out to me was that there seems to be a place where the body color is being set to #222 (which is neither the #212121 that it was nor the #1b1b1b it should be). It is overridden many times over but, can we clean that cruft up a bit either now or in future? I worry it could seep out somewhere.

Screenshot 2024-01-10 at 12 30 04 PM

**Pull request merged**